flink-ml-python/pyflink/examples/ml/recommendation/swing_example.py

```diff
@@ -0,0 +1,72 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# Simple program that creates a Swing instance and gives recommendations for items.
+
+from pyflink.common import Types
+from pyflink.datastream import StreamExecutionEnvironment
+from pyflink.table import StreamTableEnvironment
+
+from pyflink.ml.recommendation.swing import Swing
+
+# Creates a new StreamExecutionEnvironment.
+env = StreamExecutionEnvironment.get_execution_environment()
+
+# Creates a StreamTableEnvironment.
+t_env = StreamTableEnvironment.create(env)
+
+# Generates input data.
+input_table = t_env.from_data_stream(
+    env.from_collection([
+        (0, 10),
+        (0, 11),
+        (0, 12),
+        (1, 13),
+        (1, 12),
+        (2, 10),
+        (2, 11),
+        (2, 12),
+        (3, 13),
+        (3, 12)
+    ],
+        type_info=Types.ROW_NAMED(
+        ['user', 'item'],
+        [Types.LONG(), Types.LONG()])
+    ))
+
+# Creates a swing object and initialize its parameters.
+swing = Swing()\
+    .set_item_col('item')\
+    .set_user_col("user")\
+    .set_min_user_behavior(1)
+
+# Transforms the data to Swing algorithm result.
+swingTable = swing.transform(input_table)
```

Comment:
`swingTable` is ambiguous and also not conform to Python naming convention. Let's use `output` instead like other tests.